rest-tests/src/test/java/io/syndesis/qe/rest/tests/IntegrationHandler.java

```diff
@@ -114,11 +114,11 @@ public void sameNameIntegrationValidation(String integrationName, String desired
             .build();
 
         log.info("Creating integration {}", integration.getName());
-        Assertions.assertThatExceptionOfType(BadRequestException.class)
+        Assertions.assertThatExceptionOfType(AssertionError.class)
```

Comment:
I did it in this way https://github.com/syndesisio/syndesis-qe/pull/1239/commits/4a5c76ae43de12512af2bb08b30dc3a7ad50bc0e however it is a part of the PR (sorry, my bad) which is not already merged (and cannot because I have to test it more)

Reply:
I see. Yes, it seems like your solution could suite the situation the test is trying to verify a bit more. I'm closing this PR

Reply:
ok, I have extracted the commit from the WIP PR and create a separate PR for it: https://github.com/syndesisio/syndesis-qe/pull/1295